src/cli/commands/summarize.py

```
@@ -0,0 +1,533 @@
+                document_id=doc_id,
+                document_title=doc_title,
+                chunks=chunks,
+            )
+        else:
+            with console.status("[bold green]Generating summaries..."):
+                summaries = summarizer.generate_all_summaries(
+                    document_id=doc_id,
+                    document_title=doc_title,
+                    chunks=chunks,
+                )
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)